conductr_cli/conduct_main.py

```diff
@@ -420,20 +419,16 @@ def run(_args=[], configure_logging=True):
             else:
                 args.command = 'conduct'
 
-            # Ensure ConductR host is not empty
+            # Set ConductR host is --host or --ip argument not set
+            # Also set the local_connection argument accordingly
             host_from_args = conduct_url.conductr_host(args)
             if not host_from_args:
-                host_from_env = host.resolve_default_host()
+                host_from_env = host.resolve_host_from_env()
                 if host_from_env:
                     args.host = host_from_env
+                    args.local_connection = False
                 else:
-                    # Configure logging so error message can be logged properly before exiting with failure
```

Comment:
It is not possible anymore that `host.resolve_default_host()` is returning `None`. It will always return a default HOST.